src/portals/index.ts

```diff
@@ -1,11 +1,11 @@
-import { SOSHotApp } from 'portals/sos/App'
-import { HOPEHotApp } from 'portals/hope/App'
+import { App as SOSApp } from 'portals/sos/App'
+import { App as HOPEApp } from 'portals/hope/App'
 
 export const app = (portal: string) => {
```

Comment:
This should work right?
```suggestion
export const app = (portal: 'SOS' | ' Hope') => {
```

Reply:
I'm not sure, it's from `useAuthenticationQuery.me.portal`